Cargo.toml

```diff
@@ -162,6 +162,7 @@ log = "0.4.21"
 metrics = "0.21.1"
 num-traits = { version = "0.2", default-features = false }
 once_cell = "1.0"
+openssl = "0.10.64"
```

Comment:
We don't want to use `openssl` for the requirement it has on the host, but `rustls` instead. You can achieve a similar syntax using `tokio_rustls`. 👍

Reply:
Example here: https://github.com/hyperium/tonic/blob/master/examples/src/tls_rustls/client.rs.